src/press/models.py

```diff
@@ -140,7 +140,7 @@ def journal_path_url(self, journal, path=None):
 
         _path = journal.code
         request = logic.get_current_request()
```

Comment:
This can now be moved inside the `if` statement below

Reply:
I meant that this line can go in the if statement: `request = logic.get_current_request()`
we still need to concatenate `_path` and `path` independantly of `settings.DEBUG` otherwise the path argument would be ignored on live installs of Janeway in path mode

Reply:
are you okay with this being on one line?
`port = logic.get_current_request().get_port()`
`request` is not used for anything else in this function other than to get the port.